### PR TITLE
Unathi Mining Guild Away Ship - Revamp and Bug Fixes

### PR DESCRIPTION
**Pull request by courierbravo**
Makes tons of changes and bug fixes to the unathi mining guild away site to dramatically improve it. Largely to bring things into line with current standards, but also to improve the experience using the site.

### CODE REVIEW

**Comment by courierbravo** [comment]
!review

**Comment by courierbravo** [comment]
Okay, that should be the last push, its ready for review. I was so determined to "get it done" at 2am, I missed a lot of really simple things.

**Comment by courierbravo** [comment]
I lied, I think its good now. If its not, so be it, maybe one of you will catch something wrong. I'm walking away from it now so you guys can actually review it.

File: maps/away/ships/hegemony/miners_guild/miners_guild_station.dm [hunk 1]
```diff
@@ -104,6 +104,14 @@
...
 	name = "shuttle control console"
 	shuttle_tag = "Miners' Guild Shuttle"
+	icon = 'icons/obj/machinery/modular_terminal.dmi'
+	icon_screen = "helm"
+	icon_keyboard = "security_key"
+	icon_keyboard_emis = "security_key_mask"
+	is_connected = TRUE
+	has_off_keyboards = TRUE
+	can_pass_under = FALSE
+	light_power_on = 1
```
Comment (DreamySkrell) [inline maps/away/ships/hegemony/miners_guild/miners_guild_station.dm:114]: make this a subtype of `/obj/machinery/computer/shuttle_control/explore/terminal` and remove all these vars

**Comment by DreamySkrell** [comment]
image
make it 255x255

**Comment by DreamySkrell** [comment]
image
check all wires

**Comment by DreamySkrell** [comment]
image
and pipes

**Comment by DreamySkrell** [comment]
image
use `/obj/effect/map_effect/window_spawner/full/shuttle/raider`

**Comment by DreamySkrell** [comment]
image
wires should not be setting d1/d2 vars

image
and especially not the dir var
this cable specifically looks correct south-to-center, but it is actually west-to-center, just rotated visually

**Comment by DreamySkrell** [comment]
image
remove this

**Comment by DreamySkrell** [comment]
image
do not set frequencies on airlocks unless absolutely required (not here)
the tags are supposed to be unique
changing frequencies when not required just makes it more confusing for future mappers

**Comment by DreamySkrell** [comment]
image
hangar blast doors are actually fake unsim walls `/turf/unsimulated/wall/fakepdoor` that cannot be destroyed/deconstructed/etc
replace with proper blast doors, just without a linked button to control them

**Comment by DreamySkrell** [comment]
image
these big tanks need consoles to control them
use `/obj/effect/map_effect/marker/large_tank` instead of setting tags and freqs manually, see example in sol yacht or scc scout

**Comment by DreamySkrell** [comment]
image
the station is missing a IFF beacon

**Comment by DreamySkrell** [comment]
image
fax machine needs `department` var set, this is what staff/cciaa see as location of where the fax is being sent from

**Comment by DreamySkrell** [comment]
image
do not have to change it for this map, but any new ships should have docking/shuttle airlock markers defined in code, and not in the map, see scc scout for a example

**Comment by DreamySkrell** [comment]
image
transit area for the shuttle is super big, only needs to go like 7 tiles from the edge of the shuttle (does not have to be 100% exact, but just shouldn't be super big like that cause it takes up actual playable area)

**Comment by DreamySkrell** [comment]
image
some vents have `level` var set, why?

**Comment by DreamySkrell** [comment]
image
double conveyor belt here - check all of them

**Comment by DreamySkrell** [comment]
image
conveyor belt on top of a machine here

**Comment by DreamySkrell** [comment]
image
some APCs have `req_access = null`, but some don't

**Comment by courierbravo** [comment]
Changes requested were made

**Review by DreamySkrell — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**